### PR TITLE
virttest/qemu_qtree.py: Update guess_type() for QtreeDev

### PR DESCRIPTION
**Pull request by spcui**
Make the get method can get the correct QtreeDisk device
on different platform.

Signed-off-by: Shuping Cui <scui@redhat.com>

id: 1211151

### CODE REVIEW

**Comment by ypu** [comment]
This update makes the function guess_type meaningless. So please check the qemu code carefully and find another solution for cover all situations we met.

File: virttest/qemu_qtree.py [hunk 1]
```diff
@@ -163,15 +163,7 @@ def add_child(self, child):
         super(QtreeDev, self).add_child(child)
 
     def guess_type(self):
-        if self.qtree['type'] == 'virtio-blk-device':
-            return QtreeDisk
-        elif ('drive' in self.qtree and
-              self.qtree['type'] != 'usb-storage' and
-              self.qtree['type'] != 'virtio-blk-pci'):
```
Comment (xutian) [inline virttest/qemu_qtree.py]: @spcui,

I think this logical looks more clearly,  expect for usb-strorage, other devices if it has backend driver, it should be a disk.

if 'driver' in self.qtree and self.qtree['type'] != 'usb-storage':
    return QtreeDisk
else:
    return QtreeDev

Comment (spcui) [inline virttest/qemu_qtree.py]: Yes, agree your comment, will update it.

**Comment by spcui** [comment]
@xutian updated, help review again, thx?

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by xutian** [comment]
looks ok to me.

**Pull request merged**